### DISCUSSION TITLE
Grids and convergence

### DISCUSSION THREAD

**Discussion by HPL001** [idaholab/moose#29538] [Q&A Modules: Phase field]
I came across a confusing situation. I am trying to reproduce some of the particle sintering phenomena in the paper by running the following input file using combined-opt under module/combined.
When the mesh is set here, the program can compute normally and converge.
```
[Mesh] # Generate mesh (7/3D X 4/3D)
  type = GeneratedMesh
  dim = 2
  nx = 70
  ny = 40
  xmin = 0
  xmax = 70.0
  ymin = 0
  ymax = 40.0
  elem_type = QUAD4
[]
```
But when the mesh is set this way, the program no longer converges and diverges.
```
[Mesh] # Generate mesh (7/3D X 4/3D)
  type = GeneratedMesh
  dim = 2
  nx = 140
  ny = 80
  xmin = 0
  xmax = 70.0
  ymin = 0
  ymax = 40.0
  elem_type = QUAD4
[]
```
As you can see, I've actually done nothing but encrypt the grid, and I'm confused as to why this is happening.

**Comment by GiudGiud** [comment]
can you please attach the log?
It's probably just a matter of selecting solver settings that work for the finer mesh problem

**Reply by HPL001** [reply]
Hello, this is the log file. I saved the whole process, the calculation converges at the beginning, then diverges at about 0.15s, and then gets farther and farther on the way of divergence.
[001.txt](https://github.com/user-attachments/files/18134124/001.txt)

**Reply by GiudGiud** [reply]
set the max linear number of iterations higher. It calls the iteration diverged when it is slowly converging, hitting a max of 30 seemingly.

you can also consider turning off the line search with `line_search = none` as some iterations were failed because of that

**Reply by GiudGiud** [reply]
this isnt great
```
    cutback_factor = 0.5
    dt = 0.001
    growth_factor = 1.1
```
it cuts down the time step by a lot when it fails, then grows it very slowly when passing. A failed time step every now and then can occur, but it s penalizing the time stepping a lot here
